trulens/nn/models/_model_base.py

```diff
@@ -172,8 +242,24 @@ def fprop(
                 as the input. If `attribution_cut` is supplied, also return the
                 cut activations.
         """
+
+        # TODO: Can attribution_cut preceed doi_cut or follow to_cut in the model?
+        # TODO: Use the defaults method for all backends.
+
         raise NotImplementedError
 
+    def _qoi_bprop_defaults(self, doi_cut, to_cut, attribution_cut):
+        """Defaults logic for qoi_bprop common across backends."""
```

Comment:
nit: same sentence fixing:

Replaces input parameters used in qoi_bprop to default values if not explicitly provided. These are common across backends.

Reply:
Done.